pkg/agent/controller/networkpolicy/packetin.go

```diff
@@ -187,3 +223,98 @@ func getPacketInfo(pktIn *ofctrl.PacketIn, ob *logInfo) error {
 	}
 	return nil
 }
+
+// rejectRequest sends reject response to the requesting client, based on the
+// packet-in message.
+func (c *Controller) rejectRequest(pktIn *ofctrl.PacketIn) error {
```

Comment:
The new design LGTM. Thanks for addressing the comments.

Reply:
NP.